README.md

```diff
@@ -1 +1,25 @@
-# gallery-service
\ No newline at end of file
+# Air Tree n Tree Reservation Service
+
+
+## API spec
+
+`GET /api/gallery/mainPictures/:roomId`
+
+### Response
+
+`String[]`
+
+------------------------------------------------------------
+
+`GET /api/gallery/allPictures/:roomId`
+
+### Response
+
+`Picture[]`
+
+## Picture:
+| Field | Type |
+|--------| ----------- |
+| imageUrl | String |
+| description | String |
+
```

Comment:
It seems like there is also some data displaying whether or not the image has been verified by Airbnb